pkg/client/didexchange/client.go

```diff
@@ -15,9 +15,11 @@ import (
 
 	"github.com/google/uuid"
 
-	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/model"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/event"
```

Comment:
I see a common folder above. Should this be didcomm/common/event & didcomm/common/service ?

Reply:
As discussed, moved messages and event relates code to "didcomm/common/service".

Also, to remove cyclical dependency, created a new [Handler function](https://github.com/hyperledger/aries-framework-go/pull/444/files#diff-75342fbf986feea5a6bb6a33d0caff76R11) and used it in [dispatcher.Service](https://github.com/hyperledger/aries-framework-go/pull/444/files#diff-6a4454074ff31ccb82bdbcc343255467R17).